src/run.ts

```diff
@@ -1,4 +1,29 @@
 #!/usr/bin/env node
-import { runCLI } from './cli'
+import process from 'node:process'
 
-runCLI()
+const debugIndex: number = process.argv.findIndex((arg) =>
+  /^--debug$/.test(arg),
+)
+
+if (debugIndex > 0) {
```

Comment:
Can we read `debug` flag from cac?

Reply:
We don't need to debug cac, only should ensure set DEBUG before tsdown runs

Reply:
Yes, so I think it is better to put it at the top of the project entrance.